device-types/SMC Networks/SMCGS8P-SMART.yaml

```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: SMC Networks
+model: SMCGS8P-SMART
+slug: smc-networks-smcgs8p-smart
+part_number: SMCGS8P-SMART
+comments: '[Datasheet](https://www.use-ip.co.uk/datasheets/442/smc_gs8p_8port_gigabit_poe_switch.pdf)'
+airflow: left-to-right
+weight: 2.06
+weight_unit: kg
+is_full_depth: false
+front_image: true
+rear_image: true
+u_height: 1
+interfaces:
+  - name: GigabitEthernet1
+    type: 1000base-t
+  - name: GigabitEthernet2
+    type: 1000base-t
+  - name: GigabitEthernet3
+    type: 1000base-t
+  - name: GigabitEthernet4
+    type: 1000base-t
+  - name: GigabitEthernet5
+    type: 1000base-t
+  - name: GigabitEthernet6
+    type: 1000base-t
+  - name: GigabitEthernet7
+    type: 1000base-t
+  - name: GigabitEthernet8
+    type: 1000base-t
+  - name: GigabitEthernet9
+    type: 1000base-t
+  - name: GigabitEthernet10
+    type: 1000base-t
+  - name: GigabitEthernet11
+    type: 1000base-t
+  - name: GigabitEthernet12
+    type: 1000base-t
+  - name: GigabitEthernet13
+    type: 1000base-t
+  - name: GigabitEthernet14
+    type: 1000base-t
+  - name: GigabitEthernet15
+    type: 1000base-t
+  - name: GigabitEthernet16
+    type: 1000base-t
+  - name: TenGigabitEthernet1
+    type: 10gbase-x-sfpp
```

Comment:
Looking at the spec sheet this is a 1G SFP port not a 10GB SFP+ port. therefore you need to use 'type: 1000base-x-sfp'